reporting_app/templates/proc_report.html

```diff
@@ -1,57 +1,106 @@
+{% macro proc_display(proc) -%}
+{# args:
+    proc (dict) - analysis_driver_proc from the Rest API.
+#}
+
+    <ul>
+        <li>Started: {{ proc._created }}</li>
+        <li>Ended:
+            {% if proc.end_date %}
+            {{ proc.end_date }}
+            {% else %}
+            running with pid: {{ proc.pid }}
+            {% endif %}
+        </li>
+        <li>
+            Status:
+            {% if proc.status %}
+            {{ proc.status }}
+            {% else %}
+            inactive
+            {% endif %}
+        </li>
+
+{%- endmacro %}
+
+
+{% macro proc_timeline(proc) -%}
+{# args:
+    proc (dict) - analysis_driver_proc from the Rest API.
+#}
+
+    <script src="../static/node_modules/vis/dist/vis.js"></script>
+    <script src="../static/node_modules/moment/moment.js"></script>
+    <link href="../static/node_modules/vis/dist/vis.css" rel="stylesheet" type="text/css" />
```

Comment:
vis.js and moment.js are referenced locally but are not part of the PR.
I can't find vis.js online but moment.js can be found at https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.13.0/locale/uk.js